default/stringtables/en.txt

```diff
@@ -2013,6 +2013,9 @@ Sets the color of the scan-line shading over planet graphics on the sidepanel.
 OPTIONS_DB_UI_PLANET_STATUS_ICON_SIZE
 Sets the size of the planet status icon.
 
+OPTIONS_DB_UI_SHOW_SIDEPANEL_STALE_BUILDING
+Enables stale buildings on the players' planet in production window.
```

Comment:
"the players' planet" meaning unclear.

is this production only?

Reply:
I suppose we don't show buildings somewhere else. Maybe just use sidepanel here too?

Reply:
they might be in the objects list